### PR TITLE
Feature/add ha sync batch size

### PR DESCRIPTION
**Pull request by ArgTang**
#125 

* Adding property to PolicyDefinition to support setting sync batch size 
* VS changed indenting based on .editorconfig
* Fix Docker connection, for running tests
https://github.com/microsoft/Docker.DotNet/issues/397#issuecomment-500186485

### CODE REVIEW

**Comment by ArgTang** [comment]
@zidad I am not too familiar with travis, do you have an idea for what could be wrong?

**Comment by zidad** [comment]
I'm not sure if we should keep the travis build in there. Seems a bit overkill now with .NET core. @luigiberrettini knows about this? Can we just get rid of the travis build or did the changes in the PR break the build?

**Comment by zidad** [comment]
@ArgTang what was the need for the changes in the docker configuration, before it seemed to run fine on travis, can you undo that and see if it works?

**Comment by ArgTang** [comment]
@zidad i can try when i get home. The reason for the change was that i was not able to run any of the tests on my local machine.

**Comment by luigiberrettini** [comment]
Having AppVeyor and Travis CI allows to build code and run tests against Docker containers on both Windows and Linux.
It would be nice to also do so on OS X but Docker support for Travis CI matrix builds was missing at the moment I created the scripts (I didn't check if this is still the case).

**Comment by ArgTang** [comment]
@zidad reverted test running changes.

File: Source/EasyNetQ.Management.Client.IntegrationTests/ManagementClientTests.cs [hunk 1]
```diff
@@ -8,138 +8,138 @@
...
-			var channels = await managementClient.GetChannelsAsync().ConfigureAwait(false);
-
-			foreach (var channel in channels)
-			{
-				Console.Out.WriteLine("channel.Name = {0}", channel.Name);
-				Console.Out.WriteLine("channel.User = {0}", channel.User);
-				Console.Out.WriteLine("channel.PrefetchCount = {0}", channel.PrefetchCount);
-			}
-		}
+    [Integration]
```
Comment (micdenny) [inline Source/EasyNetQ.Management.Client.IntegrationTests/ManagementClientTests.cs:11]: please fix all this spaces? did you add tabs instead? please revert on what was before, to see only your changes

File: Source/EasyNetQ.Management.Client.Tests/Model/PolicySerializationTests.cs [hunk 2]
```diff
@@ -126,7 +126,8 @@ public void Should_write_all_ha_policy_without_param()
             }, ManagementClient.Settings);
             Console.WriteLine(serializedMessage);
             Assert.DoesNotContain("ha-params", serializedMessage);
+            Assert.DoesNotContain("ha-sync-batch-size", serializedMessage);
             Assert.DoesNotContain("federation_upstream_set", serializedMessage);
         }
     }
-}
\ No newline at end of file
+}
```
Comment (micdenny) [inline Source/EasyNetQ.Management.Client.Tests/Model/PolicySerializationTests.cs:133]: what's the change here?

File: Source/EasyNetQ.Management.Client.sln [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿
+
```
Comment (micdenny) [inline Source/EasyNetQ.Management.Client.sln:1]: what's the change here?

**Review by micdenny — CHANGES_REQUESTED** [review]
thank you for this PR, please check the comments

File: Source/EasyNetQ.Management.Client.IntegrationTests/ManagementClientTests.cs (hunk 1 above)
Comment (micdenny) [reply Source/EasyNetQ.Management.Client.IntegrationTests/ManagementClientTests.cs:11]: maybe it's some spacing fixed after, BTW please resolve the conflict

**Comment by Pliner** [comment]
Fixed in #172

**Pull request closed without merging**